### PR TITLE
Apply contained CCD component's tagname to container's data-selector

### PR DESCRIPTION
**Pull request by serlingpa**
https://tools.hmcts.net/jira/browse/EUI-131

This branch adds a `data-selector` attribute to the connector component's element with a value of the tag name of the contained CCD component.

### CODE REVIEW

**Comment by Mo-Lala84** [comment]
Approved. Good work Mark

File: src/app/containers/exiu-ccd-connector-wrapper/exui-ccd-connector.component.ts [hunk 1]
```diff
@@ -17,14 +25,15 @@ import {Store} from '@ngrx/store';
   `
 })
 export class ExuiCcdConnectorComponent implements AfterContentInit, OnDestroy {
-
   @ContentChild('ccdComponent') ccdComponent;
+  @ContentChild('ccdComponent', {read: ElementRef}) ccdComponentElementRef: ElementRef;
```
Comment (fazanki) [inline src/app/containers/exiu-ccd-connector-wrapper/exui-ccd-connector.component.ts]: you got double ContentChild decorator... no need to reference it twice. Think you should be able to extend line 28

**Comment by fazanki** [comment]
would be good to update comments to match your extension and add a little unit test for hostBindingValue

File: src/app/containers/exiu-ccd-connector-wrapper/exui-ccd-connector.component.ts (hunk 1 above)
Comment (serlingpa) [reply src/app/containers/exiu-ccd-connector-wrapper/exui-ccd-connector.component.ts]: Unfortunately, we need both. We need both a reference to both the HTML element and the component instance itself.

**Comment by Mo-Lala84** [comment]
Fine to commit :-)

**Pull request merged**